### PR TITLE
Regaining Titles for Listing Pages

### PR DESCRIPTION
**Pull request by dbltoe**
Somehow, in the midst of my ineptitude with GitHub, these changes got dropped. Unfortunately, I did not notice until I checked Lighthouse after upgrading my test site with all the latest files

### CODE REVIEW

**Comment by proseLA** [comment]
do not modify anything in the header/stamp area.  

there is no alpha2 release.  

anyone can look at the blame and see the history of the file.

stamping the header portion of the file is an automated process the maintainers do prior to a new release.

**Comment by dbltoe** [comment]
Wasn't blaming, just trying to establish a reference point and if alpha2 was good enough for lat9 in https://github.com/zencart/zencart/pull/5215/files#diff-067db5f9774804c3d45e98afb801c86c757dcac328b8d14ddc8766188586090d, I figured I would go with that.

**Comment by proseLA** [comment]
not sure what you are talking about.

when looking at a file, there is a button called blame on the top right:

image

if you were to click on that button, you can then see the history of the file including who modified any particular line:

https://github.com/zencart/zencart/blame/v158/includes/modules/featured_products.php

the stamp that you see at the top is done prior to a release by an automated program written by zcwilt.  you can see the commit for the stamp prior to the alpha release here:

https://github.com/zencart/zencart/commit/532a5904f0bab3f69c9a3703f7bc3593a54894a9

apparently the alpha2 release was done 15 days ago.  see:

https://github.com/zencart/zencart/tags

the stamp was done here:

https://github.com/zencart/zencart/commit/b33bf3888b0f7b660d0f7747c5901ff968eca5ab

i stand by my comments of not modifying that header information.  the alpha2 release was 15 days ago.

you modifying that header information implies that file was part of that release.  it was not.

**Comment by zcwilt** [comment]
@proseLA brings up a good point

Should someone making a commit update the header code.

There's no reason not to, if you understand the format, then updating reduces the amount of work the automated process has to do.

However unless you do it correctly, you are just wasting your time as it will eventually be overwritten.

I've always thought we need a conversation around what we put in the headers. something for another day.

**Comment by dbltoe** [comment]
Pointing out ways to accomplish a goal are always better than just pointing out errors.
I look forward to instructional information.

**Comment by scottcwilson** [comment]
This thread is quite confusing (at least to me) - so just to be sure anyone reading understands how the header blocks work:

This block at the top of each file: 

```
 * @copyright Copyright 2003-2022 Zen Cart Development Team
 * @copyright Portions Copyright 2003 osCommerce
 * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
 * @version $Id: torvista 2022 Feb 18 Modified in v1.5.8-alpha $
```

does not need to be hand-edited as long as it is present.  It gets updated when we do a release. 

If you're building a new file, then please add it (and it will get updated in the next release).

**Comment by zcwilt** [comment]
https://github.com/zencart/zencart/discussions/5216

**Comment by scottcwilson** [comment]
Why are we hardcoding strings like "Main Store Logo" and "Product ID"?

File: includes/modules/featured_products.php [hunk 1]
```diff
@@ -68,7 +68,7 @@
...
+    'text' => (($featured_products->fields['products_image'] == '' and PRODUCTS_IMAGE_NO_IMAGE_STATUS == 0) ? '' : '<a href="' . zen_href_link(zen_get_info_page($featured_products->fields['products_id']), 'cPath=' . $productsInCategory[$featured_products->fields['products_id']] . '&products_id=' . $featured_products->fields['products_id']) . '" title="' . "Product ID " . $featured_products->fields['products_id'] . '">' . zen_image(DIR_WS_IMAGES . $featured_products->fields['products_image'], $featured_products->fields['products_name'], IMAGE_FEATURED_PRODUCTS_LISTING_WIDTH, IMAGE_FEATURED_PRODUCTS_LISTING_HEIGHT) . '</a><br>') . '<a href="' . zen_href_link(zen_get_info_page($featured_products->fields['products_id']), 'cPath=' . $productsInCategory[$featured_products->fields['products_id']] . '&products_id=' . $featured_products->fields['products_id']) . '" title="' . "Product ID " . $new_products->fields['products_id'] . '">' . $featured_products->fields['products_name'] . '</a><br>' . $products_price);
```
Comment (scottcwilson) [inline includes/modules/featured_products.php:71]: Let's not hardcode English language strings.

**Review by scottcwilson — CHANGES_REQUESTED** [review]

**Comment by scottcwilson** [comment]
Closing this PR which is now a mix of various things.  Let's built smaller PRs so we can get them processed more easily.

**Pull request closed without merging**